### PR TITLE
Adopts landing page according to new catalog features

### PR DESCRIPTION
**Pull request by msa0311**
Bildschirmfoto 2021-06-14 um 16 54 03

Signed-off-by: Mario Sarcher <mario@sarcher.de>

### CODE REVIEW

File: src/renderer/initializers/welcome-menu-registry.ts [hunk 1]
```diff
@@ -19,22 +19,17 @@
...
+        title: "Browse Clusters",
         icon: "view_list",
-        click: () => navigate(catalogURL())
-      },
-      {
-        title: "Configure Preferences",
-        icon: "settings",
-        click: () => navigate(preferencesURL())
+        click: () => navigate(catalogURL({ params: { group: "entity.k8slens.dev", kind: "KubernetesCluster" } } ))
       }
```
Comment (Nokel81) [inline src/renderer/initializers/welcome-menu-registry.ts:33]: Do we not want this here anymore?

Comment (msa0311) [reply src/renderer/initializers/welcome-menu-registry.ts:33]: nope, we. decided to only have one call to action that is the most important thing for the user to do right now

Comment (Nokel81) [reply src/renderer/initializers/welcome-menu-registry.ts:33]: Okay

File: src/renderer/components/+welcome/welcome.tsx [hunk 2]
```diff
@@ -34,10 +34,11 @@ export class Welcome extends React.Component {
         <div className="box">
           <Icon svg="logo-lens" className="logo" />
 
-          <h2>Welcome to {productName} 5 Beta!</h2>
+          <h2>Welcome to {productName} 5!</h2>
```
Comment (Nokel81) [inline src/renderer/components/+welcome/welcome.tsx:37]: This shouldn't be static, it should grab the version from the package.json

Comment (msa0311) [reply src/renderer/components/+welcome/welcome.tsx:37]: lets do this in a follow up PR... This is just for UX

**Review by jakolehm — APPROVED** [review]

**Review by Nokel81 — APPROVED** [review]

**Pull request merged**